examples/CommonInterfaces/CommonRigidBodyBase.h

```diff
@@ -75,7 +75,12 @@ struct CommonRigidBodyBase : public CommonExampleInterface
 	{
 		if (m_dynamicsWorld)
 		{
-			m_dynamicsWorld->stepSimulation(deltaTime);
+			const btScalar lowestExpectedFrameRate = 10.0;
+			const btScalar subStepFrequency = 60.0;
+			const int maxSubSteps = int(subStepFrequency / lowestExpectedFrameRate);
+			const btScalar subStepDuration = 1.0 / subStepFrequency;
+
+			m_dynamicsWorld->stepSimulation(deltaTime, maxSubSteps, subStepDuration);
```

Comment:
This is how I reason through what should be the `subStepDuration` and `maxSubSteps`, so I'm adding it here to help clarify for future devs.